### ISSUE TITLE
MS Ajax CDN server does not support woff2

### ISSUE THREAD

**Issue by peterblazejewicz** [aspnet/Templates#333]
TLDR;

Please add WOFF2 mime type support to MS Ajax CDN IIS server configuration file.

Long: 

Templates now heavily rely on MS Ajax CDN network to serve static resources for Production/Staging environments. The Bootstrap used in Templates is now shipping in updated 3.3.5. A year ago BS added support for WOFF2 font type:
https://github.com/twbs/bootstrap/commit/c8855a7ad120b37fd9a25125f90093478656362f
https://gist.github.com/sergejmueller/cf6b4f2133bcb3e2f64a
http://caniuse.com/#search=woff2

The problem with serving BS from MS Ajax CDN is that IIS server (I assume) is not yet configured to serve that resource type in config.

```
➜  ~  curl -I http://ajax.aspnetcdn.com/ajax/bootstrap/3.3.5/fonts/glyphicons-halflings-regular.woff2 
HTTP/1.1 503 Service Not Available
Content-Type: text/html
Date: Sun, 22 Nov 2015 16:11:20 GMT
Expires: Sun, 22 Nov 2015 16:11:20 GMT
Mime-Version: 1.0
Server: AkamaiGHost
X-CCC: DE
X-CID: 2
Content-Length: 176
```

```
➜  ~  curl -I https://ajax.aspnetcdn.com/ajax/bootstrap/3.3.5/fonts/glyphicons-halflings-regular.woff2
HTTP/1.1 503 Service Not Available
Content-Type: text/html
Date: Sun, 22 Nov 2015 16:11:26 GMT
Expires: Sun, 22 Nov 2015 16:11:26 GMT
Mime-Version: 1.0
Server: AkamaiGHost
X-CCC: DE
X-CID: 2
Content-Length: 177
```

For comparison, the same WOFF2 font served from Bootstrap CDN:

```
➜  ~  curl -I http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/fonts/glyphicons-halflings-regular.woff2 
HTTP/1.1 200 OK
Date: Sun, 22 Nov 2015 16:07:44 GMT
Content-Type: application/octet-stream
Content-Length: 18028
Connection: keep-alive
Last-Modified: Tue, 16 Jun 2015 16:29:53 GMT
ETag: "448c34a56d699c29117adc64c43affeb"
Server: NetDNA-cache/2.2
Expires: Wed, 16 Nov 2016 16:07:44 GMT
Cache-Control: max-age=31104000
Vary: Accept-Encoding
Access-Control-Allow-Origin: *
X-Hello-Human: You should work for us! Email: jdorfman+theheader@maxcdn.com or @MaxCDNDeveloper on Twitter
X-Cache: HIT
Accept-Ranges: bytes
```

```
➜  ~  curl -I https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/fonts/glyphicons-halflings-regular.woff2
HTTP/1.1 200 OK
Date: Sun, 22 Nov 2015 16:07:51 GMT
Content-Type: application/octet-stream
Content-Length: 18028
Connection: keep-alive
Last-Modified: Tue, 16 Jun 2015 16:29:53 GMT
ETag: "448c34a56d699c29117adc64c43affeb"
Server: NetDNA-cache/2.2
Expires: Wed, 16 Nov 2016 16:07:51 GMT
Cache-Control: max-age=31104000
Vary: Accept-Encoding
Access-Control-Allow-Origin: *
X-Hello-Human: You should work for us! Email: jdorfman+theheader@maxcdn.com or @MaxCDNDeveloper on Twitter
X-Cache: HIT
Accept-Ranges: bytes
```

and WOFF served from MS Ajax CDN (which seems to be configured correctly):

```
➜  ~  curl -I https://ajax.aspnetcdn.com/ajax/bootstrap/3.3.5/fonts/glyphicons-halflings-regular.woff 
HTTP/1.1 200 OK
Accept-Ranges: bytes
Cache-Control: public, max-age=31532886
Content-Type: application/octet-stream
Date: Sun, 22 Nov 2015 16:11:39 GMT
Etag: "0d59ea787a8d01:0"
Last-Modified: Tue, 16 Jun 2015 22:56:18 GMT
P3P: CP="ALL IND DSP COR ADM CONo CUR CUSo IVAo IVDo PSA PSD TAI TELo OUR SAMo CNT COM INT NAV ONL PHY PRE PUR UNI"
Server: ECAcc (fcn/409F)
Strict-Transport-Security: max-age=300
VTag: 438262157200000000
X-Cache: HIT
X-CCC: DE
X-CID: 2
X-Powered-By: ASP.NET
X-Powered-By: ARR/2.5
X-Powered-By: ASP.NET
Content-Length: 23424
```

```
➜  ~  curl -I http://ajax.aspnetcdn.com/ajax/bootstrap/3.3.5/fonts/glyphicons-halflings-regular.woff 
HTTP/1.1 200 OK
Accept-Ranges: bytes
Cache-Control: public, max-age=31448136
Content-Type: application/octet-stream
Date: Sun, 22 Nov 2015 16:11:46 GMT
Etag: "0d59ea787a8d01:0"
Last-Modified: Tue, 16 Jun 2015 22:56:18 GMT
P3P: CP="ALL IND DSP COR ADM CONo CUR CUSo IVAo IVDo PSA PSD TAI TELo OUR SAMo CNT COM INT NAV ONL PHY PRE PUR UNI"
Server: ECAcc (fcn/40BF)
VTag: 438958256000000000
X-Cache: HIT
X-CCC: DE
X-CID: 2
X-Powered-By: ASP.NET
X-Powered-By: ARR/2.5
X-Powered-By: ASP.NET
Content-Length: 23424
```

The file bug link on MS Ajax CDN just links to MS VS connect filling bug madness.

Thanks!

**Comment by rustd** [comment]
This is not an issue with templates. I will follow up with CDN.
